### ISSUE TITLE
SFML 2.5.1-2 doesn't install any DLLs

### ISSUE THREAD

**Issue by tntxtnt** [msys2/MINGW-packages#4976]
```
Packages (1) mingw-w64-x86_64-sfml-2.5.1-2

Total Download Size:    1.60 MiB
Total Installed Size:  12.12 MiB
Net Upgrade Size:       0.00 MiB
```
Download size is too small (?), it doesn't have any binary files in it.

I tried to build it manually with `MINGW_INSTALLS=mingw64 makepkg-mingw -sLf` but an error occured:
```
H:/msys2_64/home/tnt/MINGW-packages/mingw-w64-sfml/src/SFML-2.5.1/include/SFML/Graphics/Shader.hpp:610: warning: parameters of member sf::Shader::setParameter are not (all) documented
H:/msys2_64/home/tnt/MINGW-packages/mingw-w64-sfml/src/SFML-2.5.1/include/SFML/Graphics/Shader.hpp:618: warning: parameters of member sf::Shader::setParameter are not (all) documented
H:/msys2_64/home/tn[ 12%] [32mBuilding CXX object src/SFML/System/CMakeFiles/sfml-system.dir/Win32/ThreadImpl.cpp.obj[0m
[ 13%] [32mBuilding CXX object src/SFML/System/CMakeFiles/sfml-system.dir/Win32/ThreadLocalImpl.cpp.obj[0m
make[2]: *** [doc/CMakeFiles/doc.dir/build.make:58: doc/CMakeFiles/doc] Segmentation fault
make[1]: *** [CMakeFiles/Makefile2:1193: doc/CMakeFiles/doc.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 14%] [32m[1mLinking CXX shared library ../../../lib/libsfml-system-2.dll[0m
[ 14%] Built target sfml-system
make: *** [Makefile:130: all] Error 2
[1m[31m==> ERROR:(B[m[1m A failure occurred in build().(B[m
[1m    Aborting...(B[m
```

**Comment by Alexpux** [comment]
@tntxtnt what you mean size is small? All libraries are present for me:
````
$ ls /mingw64/bin | grep sfml
libsfml-audio-2.dll
libsfml-graphics-2.dll
libsfml-network-2.dll
libsfml-system-2.dll
libsfml-window-2.dll
````

**Comment by tntxtnt** [comment]
Ah so SFML DLLs prefixed `lib` to their name, that means I have to recompile my old executables.

I have an old game and needed to copy missing SFML DLLs `sfml-xxxx-2.dll`. When I looked for them in `/mingw64/bin` I couldn't find them. I entered `sfml` in Windows explorer search bar and it couldn't find any so I assumed SFML DLLs were missing. Sorry, I should have used `grep` instead...

![image](https://user-images.githubusercontent.com/1782258/54127000-cf361080-43d6-11e9-8a89-c07d3bdeb0df.png)

![image](https://user-images.githubusercontent.com/1782258/54127054-ef65cf80-43d6-11e9-9146-704915577ddf.png)
